src/site/stages/build/plugins/check-accessibility/index.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-console, no-await-in-loop */
```

Comment:
Is the `no-await-in-loop` exception needed? Doesn't look like we're doing an `await` in a loop, right?

Reply:
Ah, I think it was at some point, but nope not anymore! Thanks for noticing that